CHANGELOG.md

```diff
@@ -1,13 +1,15 @@
 # Changelog
 
-## [~] 
+## [~]
 
 ## [0.0.1] - 08.03.2019
-- Fix: fix leaflet-path-drag dependency
-- Fix: position action bar at the right bottom of the map
+
+-   Fix: fix leaflet-path-drag dependency
+-   Fix: position action bar at the right bottom of the map
```

Comment:
Improvement: could you please add a verb here , the commit sentence sound a bit not right :)

Reply:
Fixed, thanks. 👍